src/status_im/ui/screens/usage_data/views.cljs

```diff
@@ -6,26 +6,32 @@
             [status-im.ui.components.common.common :as components.common]
             [status-im.ui.components.react :as react]
             [status-im.ui.components.status-bar.view :as status-bar]
-            [status-im.ui.screens.usage-data.styles :as styles]))
+            [status-im.ui.screens.usage-data.styles :as styles]
+            [status-im.utils.utils :as utils]))
 
 (views/defview usage-data []
   (views/letsubs [next [:get-screen-params]]
     [react/view {:style styles/usage-data-view}
      [status-bar/status-bar {:flat? true}]
-     [react/view {:style styles/logo-container}
-      [components.common/logo styles/logo]
+     [react/view {:style styles/image-container}
       [react/image {:source (:analytics-image resources/ui)
                     :style  styles/usage-data-image}]]
      [react/text {:style styles/help-improve-text}
       (i18n/label :t/help-improve)]
      [react/view
       [react/text {:style styles/help-improve-text-description}
        (i18n/label :t/help-improve-description)]]
-     [react/view styles/buttons-container
-      [components.common/button {:style    {:flex-direction :row}
-                                 :on-press #(re-frame/dispatch [:help-improve-handler true next])
+     [react/text {:style styles/learn-what-we-collect-link
+                  :on-press #(re-frame/dispatch [:open-browser {:url "https://wiki.status.im/Help_Improve_Status"}])}
```

Comment:
Maybe link here https://wiki.status.im/Help_Improve_Status#Help_Improve_Status
Which will not show the wiki header/chrome.

I also notice that tapping "read more" will then add the webpage to the Home screen as a item. It shouldn't do this.

Is there any difference/advantage in showing the system webview instead of our browser?

Reply:
Don't you mean the other way around?
Currently, it shows the Status browser (see your screenshot). And it adds this as a "bookmark" to the Home tab.

I'd be happy with the system webview.